hack/run-tests.sh

```diff
@@ -4,9 +4,9 @@ set -e
 
 source hack/common.sh
 
-prefix=kubevirt
-tag=v0.7.0
-kubeconfig=~/.kube/config
+prefix=${DOCKER_PREFIX:-kubevirt}
+tag=${DOCKER_TAG:-v0.7.0}
+kubeconfig=${KUBECONFIG:-~/.kube/config}
 
-${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -test.timeout 60m
-${TESTS_OUT_DIR}/cdi.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -test.timeout 60m
+${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix ${FUNC_TEST_ARGS}
```

Comment:
It is not very good idea to share same variable for both test-suites.
For example if I pass `--junit-output=junit.xml` and pass it to both, the result will get overwritten by each other :-/

Reply:
done